.github/workflows/pull_request.yml

```diff
@@ -18,7 +18,9 @@ jobs:
 
     steps:
       - name: ⬇️ Set up code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
```

Comment:
What's the reason for using a non-default option here? Just trying to simplify logs?